pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-7183-equation-question.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="Dmitrii Eliseev" id="DDP-7183-add-equation-question-table">
+        <createTable tableName="equation_question">
+            <column name="question_id" type="bigint">
+                <constraints primaryKey="true" primaryKeyName="equation_question_pk"/>
+            </column>
+            <column name="expression" type="text">
+                <constraints nullable="false" />
+            </column>
+            <column name="maximum_decimal_places" type="bigint" defaultValue="0" />
```

Comment:
After our [discussions](https://github.com/broadinstitute/ddp-study-server/pull/1226/files#r828977353) during the last PR, do you still feel that `0` is a valid default for this field?

My perspective here is defaulting to `0` would require a user to explicitly set the `maximumDecimalPlaces` to `null` in order to have the question present a DecimalNumber how it received it (unless the front-end is special-casing `0`), and that seems like unwieldy and unnatural behavior for the study builders to contend with.

Reply:
I agree. Haven't noticed copy-paste issue. Will remove default value.